src/locales/en.json

```diff
@@ -336,6 +336,11 @@
       "reload": "Refresh now"
     }
   },
+  "move_modal": {
+    "title": "Successful moving",
+    "text": "The moving of your data from %{from} has succeed.",
```

Comment:
Fr lang has not been changed ?

s/has succeed/has succeeded

Reply:
Not yet, I am waiting for an official wording